Latest version of plugin mapkubeapis (v0.5.1) is failing with "[signal SIGSEGV: segmentation violation code=0x1 addr=0x280 pc=0x5df11a]"

I noticed since the **mapkubeapis v.0.5.1** was released while running the command:
`helm mapkubeapis --help`

The following error appears:
```shell
[root@test-ubi8 src]# helm plugin list
NAME            VERSION DESCRIPTION
mapkubeapis     0.5.1   Map release deprecated Kubernetes APIs in-place

[root@test-ubi8 src]# helm mapkubeapis --help
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x280 pc=0x5df11a]

goroutine 1 [running]:
github.com/spf13/cobra.(*Command).ExecuteC(0x0?)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1055 +0x1a
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
main.main()
        /home/runner/work/helm-mapkubeapis/helm-mapkubeapis/cmd/mapkubeapis/map_kube_apis.go:26 +0x59
Error: plugin "mapkubeapis" exited with error
```
I've tested the above plugin with 2 versions of helm, and I get the same error: helm v3.12.3 & **helm v3.15.4** (latest) !

When I tested a previous version of this plugin like v0.5.0 which is actually downloading **v0.4.1** instead (**so there's no downloadable v0.5.0 ?!**), the above error is not appearing anymore:
```shell
[root@test-ubi8 src]# helm plugin install https://github.com/helm/helm-mapkubeapis  --version v0.5.0
Downloading and installing helm-mapkubeapis v0.4.1 ...
https://github.com/helm/helm-mapkubeapis/releases/download/v0.4.1/helm-mapkubeapis_0.4.1_linux_amd64.tar.gz
Installed plugin: mapkubeapis
[root@test-ubi8 src]# helm plugin list
NAME            VERSION DESCRIPTION
mapkubeapis     0.4.1   Map release deprecated Kubernetes APIs in-place
[root@test-ubi8 src]# helm mapkubeapis --help
Map release deprecated or removed Kubernetes APIs in-place

Usage:
  mapkubeapis [flags] RELEASE

Flags:
      --dry-run               simulate a command
  -h, --help                  help for mapkubeapis
      --kube-context string   name of the kubeconfig context to use
      --kubeconfig string     path to the kubeconfig file
      --mapfile string        path to the API mapping file (default "/root/.local/share/helm/plugins/helm-mapkubeapis/config/Map.yaml")
      --namespace string      namespace scope of the release
```

--- a/cmd/mapkubeapis/map.go
+++ b/cmd/mapkubeapis/map.go
@@ -65,10 +65,6 @@ func newMapCmd(_ io.Writer, args []string) *cobra.Command {
 
 	flags := cmd.PersistentFlags()
 	flags.ParseErrorsWhitelist.UnknownFlags = true
-	err := flags.Parse(args)
-	if err != nil {
-		return nil
-	}
 
 	settings = new(EnvSettings)
 

--- a/cmd/mapkubeapis/map.go
+++ b/cmd/mapkubeapis/map.go
@@ -41,7 +41,7 @@ var (
 	settings *EnvSettings
 )
 
-func newMapCmd(_ io.Writer, args []string) *cobra.Command {
+func newMapCmd(_ io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:          "mapkubeapis [flags] RELEASE",
 		Short:        "Map release deprecated or removed Kubernetes APIs in-place",

--- a/cmd/mapkubeapis/map_kube_apis.go
+++ b/cmd/mapkubeapis/map_kube_apis.go
@@ -21,7 +21,7 @@ import (
 )
 
 func main() {
-	mapCmd := newMapCmd(os.Stdout, os.Args[1:])
+	mapCmd := newMapCmd(os.Stdout)
 
 	if err := mapCmd.Execute(); err != nil {
 		os.Exit(1)

